.pre-commit-config.yaml

```
@@ -37,31 +37,31 @@ repos:
     ]
 
 - repo: https://github.com/PyCQA/isort
-  rev: 5.13.2
+  rev: 6.0.1
   hooks:
   - id: isort
 
 - repo: https://github.com/psf/black-pre-commit-mirror
-  rev: 24.4.2
+  rev: 25.1.0
   hooks:
     - id: black
       language_version: python3
 
 - repo: https://github.com/asottile/blacken-docs
-  rev: 1.18.0
+  rev: 1.19.1
   hooks:
   - id: blacken-docs
     additional_dependencies: [black]
 
 - repo: https://github.com/codespell-project/codespell
-  rev: v2.3.0
+  rev: v2.4.1
   hooks:
   - id: codespell
     args: [-w]
```

I liked the change in https://github.com/abravalheri/ini2toml/pull/114/files#diff-63a9c44a44acf85fea213a857769990937107cf072831e1a26808cfde9d096b9R60, should we incorporate it here?

```suggestion
    args: [--write-changes]
```